dev/ProgressRing/ProgressRing.xaml

```diff
@@ -20,8 +20,24 @@
                     <Grid x:Name="LayoutRoot" Background="Transparent">
                         <VisualStateManager.VisualStateGroups>
                             <VisualStateGroup x:Name="CommonStates">
-                                <VisualState x:Name="Active" />
-                                <VisualState x:Name="Inactive" />
+                                <VisualState x:Name="Active">
+                                    <Storyboard>
+                                        <DoubleAnimation
+                                            Storyboard.TargetName="IndeterminateAnimatedVisualPlayer"
```

Comment:
>IndeterminateAnimatedVisualPlayer [](start = 67, length = 33)

Can we target the LayoutRoot's opacity ?

Reply:
Actually this is probably better since with the determinate part, we might not want to do the same thing.